packages/fcl/src/current-user/normalize/polling-response.js

```diff
@@ -13,6 +13,10 @@ import {normalizeFrame} from "./frame"
 // }
 export function normalizePollingResponse(resp) {
   if (resp == null) return null
+  if ((!!resp.addr, !!resp.services)) {
+    resp.status = "APPROVED"
+    resp.data = {...resp}
+  }
```

Comment:
Might make sense to clone it and then create a new `resp` setting the fields as needed. This ends up being a bit recursive, and while not technically a problem, could lead to leaking some stuff to other parts of the app.

Reply:
```javascript
if (___) {
  resp = { status: "APPROVED", data: {...resp} }
}
```

Reply:
What do you think best when checking `resp` for for authn data?

- (!!resp.addr || !!resp.services)
- (!!resp.addr && !!resp.services)
- (!!resp.services) _is services sufficient?_

Reply:
I think addr and services are the defining features here, so i think what you have is fine.

Reply:
Settled on (!!resp.addr || !!resp.services)
Should be G2G